server/src/main/java/org/elasticsearch/script/field/ScriptFieldSource.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.script.field;
+
+/** This interface is used to mark classes that generate fields for a script. */
```

Comment:
Can you provide more details or context in this comment?

> generate fields for a script

Since there are so many things called "fields", it's hard to understand this without more context.

Reply:
Updated with appropriate links to the actual classes.